portfolio/src/main/java/com/google/sps/servlets/Event.java

```diff
@@ -11,8 +11,10 @@
     private final Date end;
     private String password;
     private final Key key;
+    private String centerCoord;
+    private String pathCoords;
 
-    public Event(String organizer, String eventName, String location, String description, Date start, Date end, Key key){
+    public Event(String organizer, String eventName, String location, String description, Date start, Date end, Key key, String centerCoord, String pathCoords){
```

Comment:
Consider making these coordinates something other than strings if you have some time later (I'm good with this for now), like a class that contains lat and long. It feels a little weird to have these be strings, especially the path since that's multiple coordinates